src/controllers/likings.controller.js

```diff
@@ -0,0 +1,23 @@
+import sendResult from '../utils/sendResult';
+import LikingService from '../services/liking.service';
+
+const LikingsController = {
+  async addLikeAccommdation(req, res) {
+    const { userData } = req;
+    const { accommodationId } = req.params;
+    const { like } = req.params;
+    let likeValue;
+    if (like === 'like') {
+      likeValue = true;
+    } else if (like === 'unlike') {
+      likeValue = false;
+    } else {
+      return sendResult(res, 400, 'Invalid parameter');
+    }
```

Comment:
We don't need this. All we need is
- If the user has not liked it before, we create a new Like instance, with status as `true`
- If the user has liked it before, we toggle the like status
- i.e change from `true` to `false` or `false` to `true`.